pip-requirements.txt

```diff
@@ -7,9 +7,11 @@ jupyter>=1.0
 scipy>=1.0
 notebook>=5.7.2
 aplpy
-spectral-cube
+git+https://github.com/radio-astro-tools/spectral-cube.git#egg=spectral-cube
```

Comment:
I think there are a number of packages here that are not needed for this tutorial. Should it be tailored to just those packages that are imported in the notebook? astropy, matplotlib, and requests though I've suggested changes that would remove request.

Reply:
Oh, I think these are for the whole `astropy-tutorials` package if I am not mistaken. The current way to specify tutorial dependencies are via the `requirements.txt` in the directory for collecting all files for each individual tutorial. So what I have done is to fix a bug in the CI build, hence the changes made.

Reply:
Got it, thanks for explaining!

Reply:
You are welcome